pkg/apis/edge/v2alpha1/edge-placementdecision.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2022 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SourceEdgePlacementLabelKey is the key of the label used in a EdgePlacementDecision.
+// to reference the EdgePlacement that it is part of the response to.
+// We use a label rather than a field because field selectors do not work
+// on a resource defined by a CRD.
+const SourceEdgePlacementLabelKey string = "edge.kubestellar.io/source-edgeplacement"
```

Comment:
remove this label.
if we use single decision object per edge placement, no need for this label since we would use the same object name as the placement.